pypeit/images/rawimage.py

```diff
@@ -17,401 +21,746 @@
 from pypeit import utils
 from pypeit.display import display
 
-from IPython import embed
-
-class RawImage(object):
+# TODO: I don't understand why we have some of these attributes.  E.g., why do
```

Comment:
There was some fight over just having `hdu` and then I think `headarr` crept in.
If you can reduce it to one or the other, go for it.

Reply:
I'm likely to punt on this for this PR, but will leave the comment.